c2rust-transpile/src/c_ast/mod.rs

```diff
@@ -486,6 +492,71 @@ impl TypedAstContext {
         self.index(resolved_typ_id)
     }
 
+    /// Extract decl of referenced function.
+    /// Looks for ImplicitCast(FunctionToPointerDecay, DeclRef(function_decl))
+    pub fn function_declref_decl(&self, func_expr: CExprId) -> Option<&CDeclKind> {
```

Comment:
```suggestion
    pub fn fn_declref_decl(&self, fn_expr: CExprId) -> Option<&CDeclKind> {
```

For consistency with `fn_decl_ty_with_declared_args`.

Reply:
Done.